packages/core/content-manager/server/src/services/uid.ts

```diff
@@ -106,8 +106,8 @@ export default ({ strapi }: { strapi: Strapi }) => ({
     const documentCount = await strapi.documents(contentTypeUID).count({
       filters: {
         [field]: value,
-        locale: locale ?? null,
       },
+      locale: locale ?? null,
```

Comment:
if i18n is undefined, and i18n is not enabled on the content type, maybe we should account for that in the document service.

In any case, why would it be affected by being undefined?

Reply:
Good point, I've modified it to just send locale as it is to the doc service. Is that what you mean?

Reply:
yep!